### ISSUE TITLE
Title block can't be removed

### ISSUE THREAD

**Issue by erral** [plone/volto#3307]
**Describe the bug**

It is not possible to remove the Title block of a page.

**Comment by avoinea** [comment]
@erral Yes you can, you need to remove it from required settings. https://github.com/eea/volto-eea-website-policy/blob/master/src/index.js#L19

**Comment by erral** [comment]
and that should not be the default setting?

**Comment by avoinea** [comment]
@erral That, indeed, should be discussed :wink: 

@sneridagh @tisto @robgietema @tiberiuichim
